### ISSUE TITLE
Adding 'unload' to Double-Click tab actions

### ISSUE THREAD

**Issue by PuppyGirlBelly** [mbnuqw/sidebery#924]
Hi, I recently switched to this extension after using Tree-Style tabs for almost a year. And I really enjoy what you've created, and it almost replicates my particular setup with minimal configuration. 

However, the one thing I am missing is the ability to unload tabs with a double click. Currently, the middle-click option allows for either closing or unloading. Which I appreciate, however I would prefer to not give up the option of easily closing a tab with a middle-click in order to easily unload a tab.

Here are the options on the current beta version I'm on (v5.0.0b31);

![image](https://user-images.githubusercontent.com/40257903/216671316-1264a89a-9257-4053-b291-f38f73a9b17a.png)

Thank you for your work, and I plan on using this extension for a very long time!

**Comment by GodKratos** [comment]
I would really like this option too. Was surprised not to see it available under the double click selection.

**Comment by the-nelsonator** [comment]
@PuppyGirlBelly @GodKratos any issues with unload on long-click (left or right) as an alternative? That's been given the thumbs-up by extension owner but unload on double-click won't be added until a tab-switch delay is added, see https://github.com/mbnuqw/sidebery/pull/2068#issuecomment-2764218479

**Comment by GodKratos** [comment]
> [@PuppyGirlBelly](https://github.com/PuppyGirlBelly) [@GodKratos](https://github.com/GodKratos) any issues with unload on long-click (left or right) as an alternative? That's been given the thumbs-up by extension owner but unload on double-click won't be added until a tab-switch delay is added, see [#2068 (comment)](https://github.com/mbnuqw/sidebery/pull/2068#issuecomment-2764218479)

I'm not sure I fully understand what the problem is but I assume the issue is that if you double-click a tab that doesnt have focus, you will gain focus on the first click, and then lose focus when it unloads so the active tab will then change depending on the users settings when closing the active tab instead of staying on the tab they had active before any clicking happened.

TreeStyleTabs has an unloader extension to handle this behaviour which has a setting for the double-click delay.

![Image](https://github.com/user-attachments/assets/efb97031-24e9-47f4-b0ac-20eb82b95a90)

I assume the purpose of that delay setting is to add the "delay before activation" functionality the dev suggested in #2068 so I think that would be a viable way to implement it as it works well there. (The effect is that you unload unfocussed tabs without the current active tab changing)

**Comment by GodKratos** [comment]
> [@PuppyGirlBelly](https://github.com/PuppyGirlBelly) [@GodKratos](https://github.com/GodKratos) any issues with unload on long-click (left or right) as an alternative? That's been given the thumbs-up by extension owner but unload on double-click won't be added until a tab-switch delay is added, see [#2068 (comment)](https://github.com/mbnuqw/sidebery/pull/2068#issuecomment-2764218479)

With regards to the long-click behaviour... I think I would have more problems accidentally triggering that when I want to drag & drop a tab to another location.

**Comment by the-nelsonator** [comment]
>With regards to the long-click behaviour... I think I would have more problems accidentally triggering that when I want to drag & drop a tab to another location.

Yeah that's fair. It looks like it doesn't count as a long-click as soon as you start dragging, but if you take too long before dragging it would trigger. However, right long-click shouldn't pose any issues (unless you're already using it for something else) right?
